console/react/package.json

```diff
@@ -58,6 +63,7 @@
     ]
   },
   "devDependencies": {
+    "@craco/craco": "^7.0.0-alpha.3",
```

Comment:
Alternative is `react-app-rewired` package, which seems to do the same thing, with pretty much the same configuration format, and apparently does not force usage of alpha version to work with react-scripts 5. (https://blog.alchemy.com/blog/how-to-polyfill-node-core-modules-in-webpack-5)

Reply:
Hi @jiridanek,

yes, in ES6 modules the easiest way is using craco or react-app-rewired

Reply:
If you prefer to use the dist version of the files (dist/rhea.min, js, or rhea.js) you need to load them as a standalone library in the publc/index.html.

1) npm uninstall rhea
2) move the dist lib to the public folder of the project.
![1](https://user-images.githubusercontent.com/79913332/164464454-d2677b18-db42-42f0-b6e1-0c44cf1b8730.png)

3) load the lib and attach it to the window object, so you can reuse it
![2](https://user-images.githubusercontent.com/79913332/164464395-d22209df-67e9-46dd-b283-585ced918ff0.png)

4) use it 
![3](https://user-images.githubusercontent.com/79913332/164464583-ed14a336-37ec-473e-aff4-4464bf41ebdb.png)